port/unix/omrsysinfo.c

```diff
@@ -581,6 +583,8 @@ static uint32_t attachedPortLibraries;
  * Both variables are related and modified in the same code-path.
  */
 static omrthread_monitor_t cgroupMonitor;
+#define OMRPORT_SYSINFO_PROC_DIR_BUFFER_SIZE 256
+#define OMRPORT_SYSINFO_STAT_FAILURE -1
```

Comment:
Such macros are defined at the beginning of the file and they don't need to be ifdef'ed since they are handled by the pre-processor. Avoid mixing such macros with struct definitions and function prototypes.